lib/rexml/attribute.rb

```diff
@@ -148,8 +148,12 @@ def to_s
     # have been expanded to their values
     def value
       return @unnormalized if @unnormalized
-      @unnormalized = Text::unnormalize( @normalized, doctype )
-      @unnormalized
+
+      if @element&.document
+        @unnormalized = Text::unnormalize(@normalized, doctype, entity_expansion_text_limit: @element.document.entity_expansion_text_limit)
+      else
+        @unnormalized = Text::unnormalize(@normalized, doctype)
+      end
```

Comment:
How about avoiding `if`?

```suggestion
      entity_expansion_text_limit =
        @element&.document&.entity_expansion_text_limit ||
        Security.entity_expansion_text_limit
      @unnormalized = Text::unnormalize(@normalized, doctype,
                                        entity_expansion_text_limit: entity_expansion_text_limit)
      @unnormalized
```

Reply:
OK
I see.